gormschema/gorm.go

```diff
@@ -44,7 +45,19 @@ func WithConfig(cfg *gorm.Config) Option {
 }
 
 // Load loads the models and returns the DDL statements representing the schema.
-func (l *Loader) Load(models ...any) (string, error) {
+func (l *Loader) Load(objs ...any) (string, error) {
```

Comment:
Why the renaming? I think it's OK to keep previous naming

Reply:
The `objs` stand for both the view and the model; they will be separated later and processed differently